README.md

```diff
@@ -9,8 +9,8 @@ A Go library for generating MPEG-DASH manifests.
 This library uses [Glide](https://github.com/Masterminds/glide) to manage it's dependencies. Please refer to the Glide documentation to see how to install glide.
 
 ```bash
-mkdir -p $GOPATH/zencoder
-cd $GOPATH/zencoder
+mkdir -p $GOPATH/github.com/zencoder
```

Comment:
Honestly, I forget the `src` in the path EVERY time I try to use $GOPATH in my shell.... you'd think I'd remember when _I'm_ trying to use it.